gradle/aarToJarConfiguration.gradle

```diff
@@ -0,0 +1,125 @@
+//file:noinspection GroovyAssignabilityCheck
+import com.android.build.api.attributes.BuildTypeAttr
+import org.gradle.work.DisableCachingByDefault
```

Comment:
can we inline this file into `paparazzi/build.gradle` and get rid of `buildSrc` altogether?  the SDK should be the only consumer of this `aarAsjar` transformation.

Reply:
yep, now its simplified